crates/noirc_evaluator/src/ssa/acir_gen.rs

```diff
@@ -14,9 +14,11 @@ use crate::ssa::context::SsaContext;
 use crate::ssa::node::Node;
 use crate::ssa::{mem, node};
 use crate::Evaluator;
-use crate::Gate;
 use crate::RuntimeErrorKind;
-use acvm::acir::circuit::gate::{Directive, GadgetCall, GadgetInput};
+use acvm::acir::circuit::directives::Directive;
+use acvm::acir::circuit::opcodes::{BlackBoxFuncCall, FunctionInput};
```

Comment:
Guillaume : BlackBoxFuncCall should be FuncCall

Kev: Or we can make FunctionInput be named BlackBoxFunctionInput to be consistent

Reply:
We can move this to an issue on acvm, since it has nothing to do with the migration